app/src/main/java/org/cru/godtools/ui/dashboard/tools/CategoryAdapter.kt

```diff
@@ -0,0 +1,42 @@
+package org.cru.godtools.ui.dashboard.tools
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.databinding.ObservableField
+import androidx.lifecycle.LifecycleOwner
+import androidx.lifecycle.Observer
+import org.ccci.gto.android.common.recyclerview.adapter.SimpleDataBindingAdapter
+import org.cru.godtools.databinding.DashboardListItemCategoryBinding
+
+class CategoryAdapter(lifecycleOwner: LifecycleOwner, private val dataModel: ToolsCategoryDataModel) :
```

Comment:
it looks like the only thing you are using out of the dataModel is the `selectedCategory`. let's just pass that LiveData into the constructor instead of the full data model